webapp/channels/src/components/advanced_text_editor/use_submit.tsx

```diff
@@ -238,70 +243,72 @@ const useSubmit = (
             channelTimezoneCount = data ? data.length : 0;
         }
 
-        if (prioritySubmitCheck(doSubmit)) {
+        if (prioritySubmitCheck(() => doSubmit(undefined, undefined, schedulingInfo))) {
             isDraftSubmitting.current = false;
             return;
         }
 
         if (memberNotifyCount > 0) {
-            showNotifyAllModal(mentions, channelTimezoneCount, memberNotifyCount);
+            showNotifyAllModal(mentions, channelTimezoneCount, memberNotifyCount, schedulingInfo);
             isDraftSubmitting.current = false;
             return;
         }
 
-        const status = getStatusFromSlashCommand(submittingDraft.message);
-        if (userIsOutOfOffice && status) {
-            const resetStatusModalData = {
-                modalId: ModalIdentifiers.RESET_STATUS,
-                dialogType: ResetStatusModal,
-                dialogProps: {newStatus: status},
-            };
+        if (!schedulingInfo) {
+            const status = getStatusFromSlashCommand(submittingDraft.message);
```

Comment:
This is more existing stuff, but I don't think I realized that this hook is the one responsible for a bunch of these clientside slash commands. I would've expected `submitCommand` to do that

Reply:
Well, it it what it is, and it works and has been working for ages.